### PR TITLE
Test

### PR DESCRIPTION
**Pull request by az3113**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: ruby
+rvm:
+  - 2.2
+  - jruby
```
Comment (lshimokawa) [inline .travis.yml]: Don't use jruby.

File: .codeclimate.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+engines:
+  rubocop:
+    enabled: true
+  brakeman:
+    enabled: true
+  eslint:
+    enabled: true
+  csslint:
+    enabled: true
+  coffeelint:
```
Comment (lshimokawa) [inline .codeclimate.yml]: Don't use CoffeeScript.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+language: ruby
+rvm:
+  - 2.2
```
Comment (moisesnandres) [inline .travis.yml]: We are using 2.4

File: .travis.yml (hunk 1 above)
Comment (moisesnandres) [inline .travis.yml]: why jruby?

File: .codeclimate.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  - Gemfile.lock
+  - "**.erb"
+  - "**.rb"
+  - "**.css"
+  - "**.coffee"
+  - "**.inc"
+  - "**.js"
+  - "**.module"
+exclude_paths:
+- config/
```
Comment (moisesnandres) [inline .codeclimate.yml]: spaces

File: .codeclimate.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      max_line_length:
+        enabled: false
+  duplication:
+    enabled: true
+    exclude_paths:
+      - "spec/"
+    config:
+      languages:
+        ruby:
+          mass_threshold: 30
```
Comment (lshimokawa) [inline .codeclimate.yml]: Remove this config, use defaults.
@moisesnandres add the Ruby engines in org-api (like Reek).

Comment (moisesnandres) [reply .codeclimate.yml]: ok, I will

**Pull request closed without merging**